app/assets/javascripts/sufia/form_add_remove_table_rows.js

```diff
@@ -0,0 +1,200 @@
+$(document).ready(function() {
+
+  // Add an empty new row to the table to take input for a new ID, when the form is posted
+  // to the server a new relationship will be established.
+  $(document).on("click", ".btn-add-row", function(event) {
```

Comment:
I'd really like to see this whole js file made into something like a jquery plugin (scoped to a particular widget on the page) rather than a bunch of global button handlers.